torch/_inductor/lowering.py

```diff
@@ -1470,6 +1470,12 @@ def randn(*args, **kwargs):
     raise AssertionError("should have been handled in replace_random.py")
 
 
+@register_lowering(inductor_prims.force_stride, type_promotion_kind=None)
```

Comment:
nit: maybe would be more faithful to this operator to make this something like `inductor_force_stride_order`, or, `inductor_force_stride_permutation` since we arent supporting generic strides here. 

not really important....